### PR TITLE
fix: avatar group shows +x when maxCount equals 0

### LINKED ISSUE #88: Avatar Group shows all avatars when maxCount equals 0
- [x] I'm sure this does not appear in [the issue list of the repository](https://github.com/arco-design/arco-design/issues)

## Basic Info 
- **System:** macOS mojave 10.14.6
- **Browser:** Chrome 95.0.4638.54
- **Reproduction Link:** https://codepen.io/masterx89/pen/NWvaoYM?editors=001

## What is expected?
set `maxCount = 0`, Avatar Group only shows an avatar with  `+x`.

## Steps to reproduce
If I want to let Avatar Group only shows an avatar with  `+x`, I have to set `maxCount = -x` or `'0'`, because when I set `maxCount = 0`, all avatars will show.

### PR DESCRIPTION
**Pull request by masterX89**
## Types of changes

- [ ] New feature
- [x] Bug fix
- [ ] Documentation change
- [ ] Coding style change
- [ ] Component style change
- [ ] Refactoring
- [ ] Performance improvement
- [ ] Test cases
- [ ] Continuous integration
- [ ] Typescript definition change
- [ ] Breaking change
- [ ] Others 

## Background and context

closes #88 

## Changelog

| Component | Changelog(CN) | Changelog(EN) | Related issues |
| --------- | ------------- | ------------- | -------------- |
|Avatar|修复 `Avatar` 组件设置 `maxCount = 0` 时头像全部展示的问题。|Fix the bug that the `Avatar` component still show all avatars when set `maxCount = 0`.|close #88|

## Checklist:

- [x] Test suite passes (`npm run test`)
- [x] Provide changelog for relevant changes (e.g. bug fixes and new features) if applicable.
- [x] Changes are submitted to the appropriate branch (e.g. features should be submitted to `feature` branch and others should be submitted to `main` branch)

### CODE REVIEW

**Comment by PengJiyuan** [comment]
Thanks

File: components/Avatar/group.tsx [hunk 1]
```diff
@@ -40,7 +41,7 @@ function Group(baseProps: PropsWithChildren<AvatarGroupProps>, ref) {
   const avatarCount = childrenArr.length;
   let avatarsToRender = childrenArr;
 
-  if (maxCount && avatarCount > maxCount) {
+  if (!isUndefined(maxCount) && avatarCount > maxCount) {
     const avatarsInPopover = childrenArr.slice(maxCount);
```
Comment (yinkaihui) [inline components/Avatar/group.tsx:45]: 没有对 `maxCount=null` 处理。 @MisterLuffy 这里是不是传入了 `maxCount ` 等于 number类型，就走这个判断 ？

Comment (MisterLuffy) [reply components/Avatar/group.tsx:45]: `isNumber(maxCount) && maxCount >= 0` 兼容性会更好一些

Comment (masterX89) [reply components/Avatar/group.tsx:45]: 我原来以为允许用户使用负数是方便程序员从后往前截取:joy:

**Comment by masterX89** [comment]
@yinkaihui @MisterLuffy 将条件修改为了`isNumber(maxCount) && maxCount >= 0`

**Review by yinkaihui — APPROVED** [review]

**Pull request merged**